### PR TITLE
[plugin.video.comingsoon.it@matrix] 1.2.1+matrix.1

### PR DESCRIPTION
**Pull request by phate89**
### Add-on details:

- **General**
  - Add-on name: Comingsoon Trailer
  - Add-on ID: plugin.video.comingsoon.it
  - Version number: 1.2.1+matrix.1
  - Kodi/repository version: matrix

- **Code location**
  - URL: https://github.com/phate89/plugin.video.comingsoon.it

Requires: https://github.com/xbmc/repo-scripts/pull/1861
  
  
Watch the shows and the complete episodes from the website www.comingsoon.it

### Description of changes:

v1.2.1
Fixed addon after page changes

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.video.comingsoon.it/addon.xml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+	<summary lang="en">Watch trailers from www.comingsoon.it</summary>
+    <summary lang="it_IT">Guarda i traiiler dei film da www.comingsoon.it</summary>
+	<summary lang="en_GB">Watch trailers from www.comingsoon.it</summary>
+    <description lang="it">Visualizza i trailer dei film in sala e in arrivo dal sito www.comingsoon.it</description>
+	<description lang="en">Watch the shows and the complete episodes from the website www.comingsoon.it</description>
+    <description lang="it_IT">Visualizza i trailer dei film in sala e in arrivo dal sito www.comingsoon.it</description>
+	<description lang="en_GB">Watch the shows and the complete episodes from the website www.comingsoon.it</description>
+	<forum>https://forum.kodi.tv/showthread.php?tid=214378</forum>
+    <website>https://github.com/phate89/plugin.video.comingsoon.it</website>
+    <email></email>
```
Comment (enen92) [inline plugin.video.comingsoon.it/addon.xml]: drop if empty

**Review by enen92 — APPROVED** [review]

**Review by enen92 — CHANGES_REQUESTED** [review]
See comment regarding the email element

**Comment by phate89** [comment]
Dropped the empty tag

File: plugin.video.comingsoon.it/addon.xml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+	<import addon="script.module.beautifulsoup4" version="4.3.1"/> 
+	<import addon="script.module.phate89" version="2.0.1"/>
+  </requires>
+  <extension point="xbmc.python.pluginsource" library="default.py">
+    <provides>video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+	<platform>all</platform>
+  	<language>it</language>
+    <summary lang="it">Guarda i traiiler dei film da www.comingsoon.it</summary>
```
Comment (enen92) [inline plugin.video.comingsoon.it/addon.xml]: it_IT

File: plugin.video.comingsoon.it/addon.xml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+	<summary lang="en">Watch trailers from www.comingsoon.it</summary>
```
Comment (enen92) [inline plugin.video.comingsoon.it/addon.xml]: en_GB

File: plugin.video.comingsoon.it/addon.xml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    <summary lang="it_IT">Guarda i traiiler dei film da www.comingsoon.it</summary>
+	<summary lang="en_GB">Watch trailers from www.comingsoon.it</summary>
+    <description lang="it">Visualizza i trailer dei film in sala e in arrivo dal sito www.comingsoon.it</description>
```
Comment (enen92) [inline plugin.video.comingsoon.it/addon.xml]: it_IT

File: plugin.video.comingsoon.it/addon.xml [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+	<description lang="en">Watch the shows and the complete episodes from the website www.comingsoon.it</description>
```
Comment (enen92) [inline plugin.video.comingsoon.it/addon.xml]: en_GB

**Review by enen92 — CHANGES_REQUESTED** [review]

**Comment by enen92** [comment]
ping @phate89

**Comment by phate89** [comment]
I dropped the old style lines and now they should be ok, thanks

**Comment by enen92** [comment]
`ERROR: Version mismatch for required dependency script.module.phate89, required: 2.0.1, Available: 1.2.1+matrix.1`

**Comment by phate89** [comment]
I fixed the wrong dependency version. thanks!

**Pull request merged**